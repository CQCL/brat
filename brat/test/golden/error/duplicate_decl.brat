inc(Int) -> Int
inc(x) = succ(x)

inc(Int) -> Int
inc(x) = succ(succ(x))
