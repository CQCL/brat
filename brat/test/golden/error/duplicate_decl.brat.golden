Error in test/golden/error/duplicate_decl.brat:
  Name clash: ["inc"]

